src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotHttpHeaders.java

```diff
@@ -36,18 +40,20 @@ public class BotHttpHeaders {
 
   private BotHttpHeaders(Multimap<String, String> headerValues) {
     Objects.requireNonNull(headerValues);
-    this.headerValues = headerValues;
+    headerValues
+        .entries()
+        .stream()
+        .forEach(e -> this.headerValues.put(e.getKey().toUpperCase(), e.getValue()) );
```

Comment:
Nit:
Usually proxies would convert header names to be all lower case. Would be good if we maintain it as all lower case.

Reply:
Sure, updated to use lower case instead.